lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
Okay, would it be satisfactory if I switch to `document.createElement 'message-line'`?